style.css

```diff
@@ -113,3 +146,570 @@ a {
+  height: 83px;
+}
+
+.card-placeholder-group {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  position: static;
+  width: 375px;
+  height: 3028px;
+  left: 0;
+  top: 307px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.featured-img {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0 24px;
+  position: static;
+  width: 375px;
+  height: 568px;
+  left: 0;
+  top: 0;
+  margin: 24px 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.img-placeholder {
+  position: static;
+  left: 24px;
+  top: 0;
+  width: 327px;
+  height: 237px;
+  background-image: url(/images/Img-Placeholder.png);
+}
+
+.right-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  position: static;
+  width: 327px;
+  height: 307px;
+  left: 24px;
+  top: 261px;
+  margin: 24px 0;
+  order: 1;
+}
+
+.right-block > h2 {
+  position: static;
+  width: 327px;
+  height: 36px;
+  left: 0;
+  top: 0;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  order: 0;
+  margin: 12px 0;
+}
+
+.supporting-text {
+  position: static;
+  width: 327px;
+  height: 147px;
+  left: 0;
+  top: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  margin: 12px 0;
+  order: 1;
+  flex-grow: 0;
+}
+
+.skills-tags {
+  display: flex;
+  flex-direction: row;
+  width: 297px;
+  height: 40px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: 10px 0;
+}
+
+.tags {
+  display: inline-block;
+  align-items: center;
+  border: 1px solid #8993a4;
+  height: 40px;
+  background: #fff;
+  box-sizing: border-box;
+  padding: 10px 12px;
+  font-weight: 500;
+  line-height: 20px;
+}
+
+.tag-1 {
+  width: 49px;
+  margin-right: 12px;
+}
+
+.tag-2 {
+  margin-right: 12px;
+  width: 56px;
+  left: 61px;
+  order: 1;
+}
+
+.tag-3 {
+  margin-right: 12px;
+  left: 129px;
+  width: 95px;
+  order: 2;
+  top: 0;
+}
+
+.tag-4 {
+  top: 0;
+  order: 3;
+  width: 61px;
+  left: 236px;
+}
+
+.button {
+  text-transform: capitalize;
+  border: none;
+  color: #fff;
+  text-align: center;
+  background: #ff6b00;
+  padding: 12px;
+  cursor: pointer;
+  transition-duration: 0.4s;
+  display: inline-block;
+  top: 259px;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  left: 0;
+  order: 3;
+}
+
+.button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #ff6b00;
+}
+
+.button:disabled,
+button[disabled] {
+  border: none;
+  background-color: #f4f1f0;
+  color: #595454;
+}
+
+#grid {
+  top: 592px;
+  left: 0;
+  order: 1;
+  margin: 24px 0;
+  width: 375px;
+  height: 2436px;
+  padding: 0 24px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(386px, 1fr));
+}
```

Comment:
- [ ] You did an amazing job using the grid but part of the project requirements was to use the grid for the work section and inside of the card you have to use flex. Kindly fix that.